rdr_service/model/genomics.py

```diff
@@ -652,3 +651,24 @@ class GenomicCloudRequests(Base):
 
 event.listen(GenomicCloudRequests, 'before_insert', model_insert_listener)
 event.listen(GenomicCloudRequests, 'before_update', model_update_listener)
+
+
+class GenomicMemberReportStates(Base):
+    """
+    Used for maintaning one-to-many relationship
+    from GenomicSetMember based on multiple report states
+    """
+
+    __tablename__ = 'genomic_member_report_states'
```

Comment:
I know this is somewhat nit-picky, but can we drop the 's' and name the table 'genomic_member_report_state'? All other tables use the singular.

Reply:
np